crates/cubecl-core/src/ir/kernel.rs

```diff
@@ -131,29 +144,45 @@ impl Elem {
                 FloatKind::BF16 => core::mem::size_of::<half::bf16>(),
                 FloatKind::F32 => core::mem::size_of::<f32>(),
                 FloatKind::F64 => core::mem::size_of::<f64>(),
+                FloatKind::Relaxed => core::mem::size_of::<f32>(),
+                FloatKind::TF32 => core::mem::size_of::<f32>(),
```

Comment:
That function can be made `const` I think. It can help rust unroll some loops probably.